code/components/net-base/include/NetBase.h

```diff
@@ -21,13 +19,15 @@ typedef SOCKET PlatformSocketType;
 #include <sys/socket.h>
 #include <netinet/in.h>
 
-typedef int PlatformSocketType;
+typedef int SOCKET;

```

Comment:
Ah. It'd make more sense to replace the `(SOCKET)` with `(PlatformSocketType)` in the yojimbo file then, instead of making this always define `SOCKET`.

Reply:
:+1: